### PR TITLE
makes the OB inbound sound prettier and more obvious 

### PR DESCRIPTION
**Pull request by MalricB**
## About The Pull Request

the only sound the OB makes by default is a **beep beep beep** on the laze/beacon, afaik there's a sound for the shell dropping if you're _very_ close to it but if you probably hear that you're dead anyway. as a consequence of this making a [very obvious sound](https://soundcloud.com/user328212364/ob-warning-announce-1), xenos might run away faster - but to compensate I've sped up the timing by 1 (one) whole second (mostly to sync it to the SFX).
there was two ways I could have went with this;  A) make the SFX + voiceover only able to be heard by marines in the entire z-level as to not disrupt xenos somehow dodging OBs easier due to it being more obvious and louder (?) but i'm running into issues and it doesn't make much sense for different SFXs to play depending on which side you're on.
or B) save me the trouble of making shitcode / pestering five plus different coders to help me as my first PR and doing this.

Edit: I do plan on making a VO-less version that'd play out for xenos, and make marines only hear the VO+SFX version in the near future.

sadly I couldn't find a way to extract planetside 2's voiceover for the OB, so the voiceover is gonna be heard alongside the SFX by anyone in proximity.
depending on maintainers I can speed up the OB SFX alongside it dropping if this causes any balancing issues (doubt it will though) by two/a second and a half more, or go with my original plan of making it only be able to be heard by marines (will require me to get help from others) 
EDIT: planetside OB is now able to be heard by all humans on the z-level, and in an area around the impact zone (intended for xenomorphs, can probably be polished)

## Why It's Good For The Game

some marines just don't have enough multitasking abilities to look at chat while they're fighting. making the OB more obvious should hopefully reduce chances of marines running into OBs.
it's also absolutely orgasmic to hear if you're next to the OB cannon when it fires. still just as hype when you're planetside but not as much.

## Changelog
:cl: MalricB, Slayer, Hughgent
soundadd: added OB_warning_announce.ogg
soundadd: added OB_firing.ogg
soundadd: added OB_warning_announce_novoiceover.ogg
code: removed alert.ogg from overwatch.dm, replaced with OB_warning_announce.ogg
balance: sped up OB by one whole second
/:cl:

### CODE REVIEW

File: code/game/objects/structures/orbital_cannon.dm [hunk 1]
```diff
@@ -161,6 +161,14 @@
 
 	update_icon()
 
+/// Handles the playing of the Orbital Bombardment incoming sound, usually a spiraling whistle noise but can be overridden.
+/obj/structure/orbital_cannon/proc/ob_pre_impact_sound(target, var/ob_sound = 'sound/effects/OB_incoming.ogg')
```
Comment (Hughgent) [inline code/game/objects/structures/orbital_cannon.dm:165]: This is a bad proc name, it does more than just play sounds on the ground, it also has an animation. It should have a more appropriate name.

File: code/game/objects/structures/orbital_cannon.dm [hunk 2]
```diff
@@ -190,13 +195,14 @@
 			inaccurate_fuel = abs(GLOB.marine_main_ship?.ob_type_fuel_requirements[4] - tray.fuel_amt)
 
 	var/turf/target = locate(T.x + inaccurate_fuel * pick(-1,1),T.y + inaccurate_fuel * pick(-1,1),T.z)
-	for(var/i in hearers(WARHEAD_FALLING_SOUND_RANGE,target))
-		var/mob/M = i
-		M.playsound_local(target, 'sound/effects/OB_incoming.ogg', falloff = 2)
+
+	playsound(target, 'sound/effects/OB_warning_announce.ogg', 100, FALSE, 20, 4) //the ground
```
Comment (Hughgent) [inline code/game/objects/structures/orbital_cannon.dm:199]: Should everyone on the ground be hearing this?

**Review by Hughgent — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/orbital_cannon.dm (hunk 2 above)
Comment (MalricB) [reply code/game/objects/structures/orbital_cannon.dm:197]: something to think about but can do if #[5315](https://github.com/tgstation/TerraGov-Marine-Corps/pull/5315) gets merged

Comment (MalricB) [reply code/game/objects/structures/orbital_cannon.dm]: and done

File: code/game/objects/structures/orbital_cannon.dm [hunk 3]
```diff
@@ -190,13 +193,15 @@
 			inaccurate_fuel = abs(GLOB.marine_main_ship?.ob_type_fuel_requirements[4] - tray.fuel_amt)
 
 	var/turf/target = locate(T.x + inaccurate_fuel * pick(-1,1),T.y + inaccurate_fuel * pick(-1,1),T.z)
-	for(var/i in hearers(WARHEAD_FALLING_SOUND_RANGE,target))
-		var/mob/M = i
-		M.playsound_local(target, 'sound/effects/OB_incoming.ogg', falloff = 2)
+
+	playsound_z_humans(2, 'sound/effects/OB_warning_announce.ogg', 100, FALSE, 20, 4) //for marines on ground
```
Comment (TiviPlus) [inline code/game/objects/structures/orbital_cannon.dm:197]: Use T.x

File: code/game/objects/structures/orbital_cannon.dm [hunk 4]
```diff
@@ -190,13 +193,15 @@
...
+	playsound(target, 'sound/effects/OB_warning_noVO.ogg', 100, FALSE, 20, 4) //for xenomorphs and anyone nearby on ground. can probably be polished to only play to xenos instead
```
Comment (TiviPlus) [inline code/game/objects/structures/orbital_cannon.dm:198]: Rename this file to more accurately reflect what it plays pls

File: code/game/objects/structures/orbital_cannon.dm [hunk 5]
```diff
@@ -190,13 +193,15 @@
...
+	playsound_z(loc.z, 'sound/effects/OB_warning_announce.ogg', 100) //the ship
```
Comment (TiviPlus) [inline code/game/objects/structures/orbital_cannon.dm:199]: You can just use z

Comment (MalricB) [reply code/game/objects/structures/orbital_cannon.dm:199]: can I? didn't seem to play the soundfile when not specifying a zlevel, i'll try it out again

File: code/game/objects/structures/orbital_cannon.dm (hunk 4 above)
Comment (MalricB) [reply code/game/objects/structures/orbital_cannon.dm]: OB_warning_announce_novoiceover.ogg sounds okay?

File: code/game/objects/structures/orbital_cannon.dm (hunk 3 above)
Comment (MalricB) [reply code/game/objects/structures/orbital_cannon.dm]: playsound_z_humans needs a z level so I can't

Comment (Hughgent) [reply code/game/objects/structures/orbital_cannon.dm]: `target.z`

Comment (MalricB) [reply code/game/objects/structures/orbital_cannon.dm]: updated accordingly, thanks hugh

**Review by TiviPlus — APPROVED** [review]

**Review by TiviPlus — APPROVED** [review]

**Review by TiviPlus — APPROVED** [review]

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-5). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**